src/reference/00-Getting-Started/07A-Task-Graph.md

```diff
@@ -0,0 +1,324 @@
+---
+out: Task-Graph.html
+---
+
+  [Basic-Def]: Basic-Def.html
+  [Scopes]: Scopes.html
+  [Make]: https://en.wikipedia.org/wiki/Make_(software)
+  [Ant]: http://ant.apache.org/
+  [Rake]: https://ruby.github.io/rake/
+
+Task graph
+----------
+
+This page explains `build.sbt` definition in more detail.
+It assumes you've read [.sbt build definition][Basic-Def] and
+[scopes][Scopes].
+
+### What's the point of the build.sbt DSL?
+
+As we saw before, a [build definition][Basic-Def] consists of subprojects
+with a map called `settings` describing the subproject.
```

Comment:
"with some `settings` describing the subproject" would be clearer I think. The fact that it's a map is not very useful, especially since it's not a `scala.Map`, nor is it any "normal" map that someone may think of since it contains tons of dependency information and other things